ps_client_frontend/src/pages/profile/Profile.jsx

```diff
@@ -0,0 +1,111 @@
+/* Framework & CSS */
+import React, { useState } from "react";
+import "./profile.css";
+import { useAppState } from "../../providers/AppStateProvider";
+import { Navigate } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+
+import RoundBtn from "../../components/roundBtn/RoundBtn";
+import userEdit from "../../images/user-pen-solid.svg";
+import back from "../../images/arrow-left-solid.svg";
+import save from "../../images/floppy-disk-solid.svg";
+
+import Input from "../../components/Input/Input";
+import UserDetail from "../../components/userDetail/UserDetail";
+import { useMutation } from "@apollo/client";
+import { EDIT_USER } from "../../hooks/mutations/userMutations";
+
+import { SET_USER } from "../../constants";
+
+export default function Profile() {
```

Comment:
at some point I'd like to put this into a reducer instead of state values, but not necessary for this pr.

Reply:
Created an issue: [Add Reducer](https://github.com/mrogers23458/plan-simply/issues/5)